test/protobuf/protoc/generator/enum_test.exs

```diff
@@ -36,10 +36,15 @@ defmodule Protobuf.Protoc.Generator.EnumTest do
 
     assert msg =~ """
              field :A, 0
+
              field :B, 1
+
              field :HAS_UNDERSCORES, 2
+
              field :HAS_UNDERSCORES_X, 3
+
              field :HAS_UNDERSCORES_, 4
+
```

Comment:
We only get the double newlines in tests, right? The generated code won't have this?